src/framework/anim/evaluator/empty-anim-track.js

```diff
@@ -0,0 +1,17 @@
+import { AnimTrack } from './anim-track.js';
+
+/**
+ * This AnimTrack can be used as a placeholder track when creating a state graph before having all associated animation data available.
+ */
+class EmptyAnimTrack extends AnimTrack {
+    /**
+     * Create a new empty AnimTrack instance.
+     *
+     * @hideconstructor
+     */
+    constructor() {
```

Comment:
this looks like you just need an instance of AnimTrack with empty data instead of a new class?

Reply:
it could be AnimTrack.EMPTY, similarly to Vec3.ZERO for example.

Reply:
Yep I think a static property on the AnimTrack class would be better, I'll update it to that.